sandbox/handlers.js

```diff
@@ -77,10 +78,12 @@ async function immunisation(req, res, next) {
             patientIdentifier: patientIdentifier
         }
     });
-    if (patientIdentifier == "9912003888") {
+    if (patientIdentifier == null || patientIdentifier == "") {
```

Comment:
It's fine here, but I would recommend reading up on equality in js, as it can easily catch you out:
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
* https://dorey.github.io/JavaScript-Equality-Table/